.github/workflows/build-test-push-workflow.yml

```diff
@@ -278,13 +278,13 @@ jobs:
         if: ${{ always() }}
         run: |
           make cluster-down
-      - name: Test Report
-        uses: dorny/test-reporter@v1
-        if: success() || failure()    # run this step even if previous step failed
-        with:
-          name: Integration Tests          # Name of the check run which will be created
-          path: inttest-*.xml         # Path to test results
-          reporter: jest-junit        # Format of test results
+      #- name: Test Report
```

Comment:
Nit: Do we need this commented? Or can we remove it?

Reply:
we need to still investigate why the test report creation fails here. i will keep it for sometime